site/pages/examples/sidebar-apps.js

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import { HtmlExample } from '../../html_example';
 import { Page } from '../../page';
 
-const basicSidebar = `<div class='flex-parent viewport-full relative overflow-hidden'>
+const basicSidebar = `<div class='flex-parent h-viewport-full relative overflow-hidden'>
   <div class='flex-child w-full w240-ml absolute static-ml left bottom'>
-    <div class='flex-parent flex-parent--column viewport-third h-full hmax-full bg-white'>
+    <div class='flex-parent flex-parent--column h-viewport-1/3 h-full hmax-full bg-white'>
```

Comment:
This example seems broken - left sidebar should be taller?

<img width="438" alt="Screen Shot 2021-04-13 at 1 02 26 PM" src="https://user-images.githubusercontent.com/108094/114613688-86f21800-9c58-11eb-813b-0a35bd0315e8.png">

The markup for these layout examples is poor. We should cut them or rewrite them to be cleaner.